recipe/meta.yaml

```diff
@@ -28,10 +29,12 @@ requirements:
 test:
   requires:
     - pip
+    - python <3.10
   commands:
     - pip check
   imports:
     - bleach
+    - bleach._vendor
```

Comment:
Packages starting with an underscore are "private" and should not be part of the import tests.

Reply:
The issue has been resolved:
https://github.com/AnacondaRecipes/bleach-feedstock/blob/update_to_4.1.0/recipe/meta.yaml#L35&L37